src/reading/parseFile.js

```diff
@@ -4,6 +4,8 @@ import mapValues from 'lodash.mapvalues'
 
 const wgs84 = gdal.SpatialReference.fromEPSG(4326)
 
+const maxLayers = 20
```

Comment:
This seems like a hack - we should figure out why it crashed over 20 instead of limiting it to 20.